### PR TITLE
make mono-run work

### LINKED ISSUE #3: make `mono-run` run locally
port of this issue closer to the code: https://github.com/openbraininstitute/prod-NGV/issues/1

After investigating this problem I noticed that the work is more than 1 week. I was able to make mono-run work locally. Julia still gives me problems. 

Thus I resized this issue to `make mono-run run locally`

### PR DESCRIPTION
**Pull request by cattabiani**
fix #3 

## Context

MultiscaleRun was heavily dependent on bb5. Its loss made the program fail in various parts. Here I want to start from the base and rebuild from the ground up checking with the `tiny_CI` circuit. The only circuit I was able to save for posterity. 

It is too slow to run on a single core. We will employ mpirun

## Scope

- remove bloodflow dependencies
- remove steps dependencies
-  fix spack installation problems
- leave julia for now, remove diffeqpy as it is problematic
- the latest release of `neurodamus-models` gives a seg-fault for glu2 and cadifus. We should use develop
- add `setup.py` to set spack, python and prepare the tiny_CI simulation
- remove ci. I will reactivate once something works. It is less important now
- update readme
- add the testing in the readme
  
 ## Testing

@jamesgking check the testing section in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+
+You can always modify them and recall `setup.sh`. It will not override your changes. 
+
+### Test
+
+You just need to go to `tiny_CI_test` and run. The simulation is too slow with just one core. I suggest at least 8 cores. Do not go above 90 for now as this leaves some cores without neurons (edge case that I did not check). 
+
+```bash
+cd tiny_CI_test
+mpirun -np 8 multiscale-run compute
```
Comment (jamesgking) [inline README.md]: Do we have some viewer that showcases the output so we see that it ran and did what was expected? Or is it too early and that will come later?

Comment (cattabiani) [reply README.md]: we have post-processing but I did not actually check it. I am still fighting with julia

File: README.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+
+The script does this:
+- set various env variables
+- create a `spackenv` folder with the necessary dependencies
+- create a `julia` project with the necessary dependencies in `juliaenv`
+- create a python virtual env in `venv` 
+- call `pip install -e .` for development
+- create the test folder `tiny_CI_test`
+- fill it with the necessary data
+
```
[the remark this answers is not in the document]
Comment (cattabiani) [reply README.md]: I do not understand. This is the "any time setup" that you need before running simulations. It does both the first time and the other time setups, skipping what is there. 

How to run the simulations is in ### Test. It is already divided
